app/partial/spatial/liveViewPanel/mapPanel/mapPanel.js

```diff
@@ -99,7 +99,8 @@ angular.module('unionvmsWeb').controller('MapCtrl',function($log, $scope, locale
                         var item = {
                             type: 'asset',
                             name: mapService.overlay.get('vesselName'),
-                            guid: assetId
+                            guid: assetId,
+                            isFishingActivity: $scope.isActivityDetailsAllowed()
```

Comment:
I think reusing a method that is meant for a different purpose is not reliable. I.e., and looking at the implementation of `isActivityDetailsAllowed()`, the item might be an activity, but for which details are not allowed, so this method returns `false`. Using `$scope.popupRecContainer.currentType === 'ers'` might be more appropriate.

Reply:
OK